docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,1001 @@
+# Tutorial: Build your first block
+
+In this tutorial, you will build a "Copyright Date Block"—a basic yet practical block that displays the copyright symbol (©), the current year, and an optional starting year. This type of content is commonly used in website footers.
+
+The tutorial will guide you through the complete process, from scaffolding the block plugin using the [`create-block`](https://developer.wordpress.org/block-editor/getting-started/devenv/get-started-with-create-block/) package to modifying each file. While previous WordPress development experience is beneficial, it's not a prerequisite for this tutorial.
+
+By the end of this guide, you will have a clear understanding of block development fundamentals and the necessary skills to create your own WordPress blocks.
+
+## What you're going to build
+
+Here's a quick look at what you're going to build.
+
+![What you're going to build](https://developer.wordpress.org/files/2023/12/block-tutorial-1.png)
+
+You can also interact with the finished project in [WordPress Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/WordPress/block-development-examples/trunk/plugins/copyright-date-block-09aac3/_playground/blueprint.json) or use the [Quick Start Guide](https://developer.wordpress.org/redesign-test/block-editor/getting-started/quick-start-guide/) to install the complete block plugin in your local WordPress environment.
```

Comment:
"Quick Start Guide" points to a `redesign-test` link

Reply:
Good catch, all fixed.